src/bastion-executor/benches/blocking.rs

```diff
@@ -15,19 +12,17 @@ use test::Bencher;
 #[bench]
 fn blocking(b: &mut Bencher) {
     b.iter(|| {
-        let handles = (0..10_000)
+        (0..100)
             .map(|_| {
                 blocking::spawn_blocking(
                     async {
-                        let duration = Duration::from_millis(1);
+                        let duration = Duration::from_millis(0);
                         thread::sleep(duration);
                     },
                     ProcStack::default(),
                 )
             })
-            .collect::<Vec<RecoverableHandle<()>>>();
-
-        run(join_all(handles), ProcStack::default());
+            .collect::<Vec<_>>()
```

Comment:
Collecting doesn't mean that they are completed.

Reply:
Yes, blocking.rs benchmarks the "spawning" and run_blocking benchmarks the running, is that ok ?